src/components/utils.js

```diff
@@ -8,3 +8,20 @@ export const isValidAddress = address => {
     return false;
   }
 };
+
+export const validateHash = input => {
+  if (input.length === 0) {
```

Comment:
Can you use a regex matcher, there is no need to tell them its too long or short, if its wrong its wrong....

Reply:
Can return only true/false